commands/builds/create.js

```diff
@@ -81,9 +88,10 @@ module.exports = {
   help: 'Create build from contents of current dir',
   description: 'create build',
   flags: [
-    { name: 'source-url', description: 'Source URL that points to the tarball of your application\'s source code', hasValue: true },
-    { name: 'tar', description: 'Path to the executable GNU tar', hasValue: true },
-    { name: 'version', description: 'Description of your new build', hasValue: true }
+    { name: 'source-url', description: 'source URL that points to the tarball of your application\'s source code', hasValue: true },
+    { name: 'tar', description: 'path to the executable GNU tar', hasValue: true },
+    { name: 'version', description: 'description of your new build', hasValue: true },
+    { name: 'include-vcs-ignore', description: 'include files ignores by VCS (.gitignore, ...) from the build' }
```

Comment:
GNU Tar on OSX does support this flag.
We don't accept BSD Tar, as it can generate invalid tarballs.

This command [detects that](https://github.com/heroku/heroku-builds/blob/master/commands/builds/create.js#L15), and fallbacks to using node's tar (which is actually BSD too).
So on OSX, if the machine has GNU Tar, that flag will be handled. If it doesnt', we won't even be calling the tar executable anyway.